src/hotspot/share/cds/cdsEnumKlass.cpp

```diff
@@ -0,0 +1,136 @@
+/*
+ * Copyright (c) 2023, Oracle and/or its affiliates. All rights reserved.
```

Comment:
Please update the copyrights on the new files.

Reply:
I changed the dates to `2023, 2024` as these files were initially published in the leyden repo last year.